conf/app.conf

```diff
@@ -2,8 +2,11 @@ appname = google-scraper
 httpaddr = 0.0.0.0
 httpport = ${PORT||8080}
 runmode = ${BEEGO_ENV||dev}
+redisUrl = ${REDIS_URL||127.0.0.1:6379}
```

Comment:
If think if we order all env variable alphabetically here, then it will be inconsistent to grouping related envs.

Like currently first 4 env(`appname`, `httpaddr`,`httpport`, `runmode` ) giving an overview how the app will be running. Then if we have some more other type env variable with name `a..`, `b..`, it will break the consistency.

And the bellow envs are similar, they I keep them together, But If we order them alphabetically, we can keep them together.  
```
crawlingJobName = crawling_job
redisEnqueuerName = google_scraper
```

What do you think?